app/frontend/helpers/distinct.ts

```diff
@@ -1 +1 @@
-export default (array) => Array.from(new Set(array))
+export default <T>(array: T[]) => Array.from(new Set(array)) as T[]
```

Comment:
such default export hampers searchability by function name which is a bit annoying at least

Reply:
Fixed.